config/src/main/java/com/alibaba/nacos/config/server/service/capacity/CapacityService.java

```diff
@@ -181,7 +149,7 @@ private void correctTenantUsage() {
             try {
                 Thread.sleep(100);
             } catch (InterruptedException e) {
-                // ignore
+                // Ignore.
```

Comment:
do not use comment to ignore, change the exception name as ignore, or ignored just be ok.